### PR TITLE
Launcher page

### PR DESCRIPTION
**Pull request by CodexAdrian**
Fixes MOD-480

### CODE REVIEW

File: assets/images/external/gdlauncher.png
Comment (brawaru) [inline assets/images/external/gdlauncher.png:1]: This icon can be made smaller (1080x1080, 26.2 KB ⇒ 270x270, 5.22 KB) without significant loss in quality.

gdlauncher

File: pages/launcher.vue [hunk 1]
```diff
@@ -0,0 +1,1852 @@
...
+</script>
+
+<template>
+  <div>
+    <div class="landing-hero">
+      <h1 class="main-header">
+        Download Modrinth <br />
+        app
+        {{
+          os ? `for ${os?.includes('Mac') ? 'Mac' : os?.contains('Win') ? 'Windows' : 'Linux'}` : ''
```
Comment (brawaru) [inline pages/launcher.vue:34]:
```suggestion
          os ? `for ${os?.includes('Mac') ? 'Mac' : os?.includes('Win') ? 'Windows' : 'Linux'}` : ''
```

`String.prototype.contains` is not a function.

File: pages/launcher.vue [hunk 2]
```diff
@@ -0,0 +1,1852 @@
...
+              <div class="cell">
+                <div class="name">Fabric API</div>
+                <div class="description">by modmuss</div>
+              </div>
+              <div class="cell important">1.0.0</div>
+              <div class="cell check">
+                <Checkbox :model-value="true" />
+                <button class="btn icon-only transparent">
+                  <TrashIcon />
+                </button>
```
Comment (brawaru) [inline pages/launcher.vue:86]: Accessibility: tab navigation interference; these are used purely presentatively and as such, should not be able to receive focus.

**Review by brawaru — COMMENTED** [review]
Just passing by leaving some comments for the future

**Review by Geometrically — APPROVED** [review]

**Pull request merged**